empire/server/modules/powershell/persistence/misc/add_sid_history.py

```diff
@@ -28,14 +28,13 @@ def generate(main_menu, module: PydanticModule, params: Dict, obfuscate: bool =
 
         script = moduleCode
 
-        # ridiculous escape format
-        groups = " ".join(['"\\""'+group.strip().strip("'\"")+'"""' for group in params["Groups"].split(",")])
-
         # build the custom command with whatever options we want
-        command = '""misc::addsid '+params["User"] + ' ' + groups
-
+        command = f'"sid::add /sam:{params["User"]} /new:{params["Group"]}"'
```

Comment:
Change "Group" to "Groups" and this should be good to go.

Reply:
Looks like there is another issue that comes back when making that change. Did you see this error?
![image](https://user-images.githubusercontent.com/20302208/132249313-6f43e9a3-1189-4177-939a-d3cfbc45c9bd.png)

Reply:
Hey, Thanks for your time! 
Regarding groups i'm not sure but I think mimikatz supports only one group at a time, correct me if im wrong.
About the second error I think that happens in parsing mimikatz response, I'll add that change as well